src/scroll-timeline-css.js

```diff
@@ -32,7 +32,7 @@ function initMutationObserver() {
    */
   function handleStyleTag(el) {
     // Don’t touch empty style tags.
```

Comment:
Can we add to this comment explaining why we skip tags with aphrodite? Maybe link to https://github.com/Khan/aphrodite/blob/master/src/inject.js to show why we check for the aphrodite attribute.

Reply:
Yep, added!